### PR TITLE
Fix python release build.

### PR DESCRIPTION
**Pull request by haon4**

### CODE REVIEW

File: kokoro/release/python/linux/build_artifacts.sh [hunk 1]
```diff
@@ -7,11 +7,13 @@ pushd $(dirname $0)/../../../..
 
 export REPO_DIR=protobuf
 export BUILD_VERSION=`grep -i "version" python/google/protobuf/__init__.py | grep -o "'.*'" | tr -d "'"`
-export BUILD_COMMIT=v$BUILD_VERSION
+export BUILD_COMMIT=master
```
Comment (TeBoring) [inline kokoro/release/python/linux/build_artifacts.sh]: If we pin to master, we need to change it when we work on release branch?

File: kokoro/release/python/linux/config.sh [hunk 2]
```diff
@@ -6,6 +6,12 @@ function pre_build {
     # Runs in the root directory of this repository.
     pushd protobuf
 
+    if [ -n "$IS_OSX" ]; then
```
Comment (TeBoring) [inline kokoro/release/python/linux/config.sh]: We probably should file a bug to multibuild and kokoro to let them include these.

File: kokoro/release/python/linux/config.sh [hunk 3]
```diff
@@ -6,6 +6,12 @@ function pre_build {
...
+      brew install libatomic
```
Comment (TeBoring) [inline kokoro/release/python/linux/config.sh]: Shall we create a separate file in kokoro/release/python/macos?

File: kokoro/release/python/linux/config.sh (hunk 2 above)
Comment (haon4) [reply kokoro/release/python/linux/config.sh]: But libatomic is something that we include recently. I assume not everyone need it, right?

File: kokoro/release/python/macos/config.sh [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+# Test for OSX with [ -n "$IS_OSX" ]
+
+function pre_build {
+    # Any stuff that you need to do before you start building the wheels
+    # Runs in the root directory of this repository.
+    pushd protobuf
+
+    export CC=gcc
+    export CXX=g++
+    # Build protoc
```
Comment (TeBoring) [inline kokoro/release/python/macos/config.sh]: No need to brew install now?

File: kokoro/release/python/linux/build_artifacts.sh (hunk 1 above)
Comment (haon4) [reply kokoro/release/python/linux/build_artifacts.sh]: Changed to extract from the KOKORO_JOB_NAME name instead.

File: kokoro/release/python/macos/config.sh (hunk 4 above)
Comment (haon4) [reply kokoro/release/python/macos/config.sh]: Still trying to verify that this actually works. It is still running...

File: kokoro/release/python/macos/build_artifacts.sh [hunk 5]
```diff
@@ -7,17 +7,23 @@ pushd $(dirname $0)/../../../..
...
 export TRAVIS_OS_NAME="osx"
 
+rm -rf artifacts/
+rm -rf multibuild/
 mkdir artifacts
 export ARTIFACT_DIR=$(pwd)/artifacts
 
 git clone https://github.com/matthew-brett/multibuild.git
-cp kokoro/release/python/linux/config.sh config.sh
+cp kokoro/release/python/macos/config.sh config.sh
```
Comment (TeBoring) [inline kokoro/release/python/macos/build_artifacts.sh]: I think you don't need to copy. It's already in the place you want to move.

Comment (haon4) [reply kokoro/release/python/macos/build_artifacts.sh]: Actually no. On L6, we change the directory to the repo root, so we are copying it to the repo's root directory.

**Comment by haon4** [comment]
Looks like setting the -std=c++14 flag on Mac OS resolve the issue.

**Pull request merged**